plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -237,34 +237,51 @@
     },
     "bls12_381_finalVerify": {
         "cpu": {
-            "arguments": 388656972,
+            "arguments": 388443360,
             "type": "constant_cost"
         },
         "memory": {
             "arguments": 1,
             "type": "constant_cost"
         }
     },
-    "bls12_381_mulMlResult": {
+    "bls12_381_millerLoop": {
         "cpu": {
-            "arguments": 2544991,
+            "arguments": 401885761,
```

Comment:
In case somebody is confused like I was, this is `bls12_381_mulMlResult` being replaced with `bls12_381_millerLoop`, not a 160x growth in cost.